admin-dev/themes/default/template/helpers/list/list_header.tpl

```diff
@@ -29,8 +29,13 @@
 			$(".ajax_table_link").click(function () {
 				var link = $(this);
 				$.post($(this).attr('href'), function (data) {
-					if (data.success == 1) {
-						showSuccessMessage(data.text);
+				  // If response comes from symfony controller
+          // then data has "status" and "message" properties
+          // otherwise if response comes from legacy controller
+          // then data has "success" and "text" properties.
+
+					if (data.success == 1 || data.status == true) {
```

Comment:
```suggestion
					if (data.success === 1 || data.status === true) {
```

But why checking both?

Reply:
Read comment above. :stuck_out_tongue:

Reply:
I've added strict type checking for `data.status` as it comes from new controller, however, I kept `==` for legacy value, because it's used across all legacy lists, so I don't want to break it. :/